src/datadoc/frontend/fields/display_dataset.py

```diff
@@ -63,6 +63,19 @@ def get_owner_options() -> list[dict[str, str]]:
     return dropdown_options
 
 
+def get_data_source_options() -> list[dict[str, str]]:
```

Comment:
Could this method be moved to display_base and imported in display_dataset and display_variables instead of two identical methods?